packages/components/src/components/validate/validate.component.ts

```diff
@@ -0,0 +1,321 @@
+ * @csspart alert__base - The container that wraps the alert.
+ * @csspart alert__message - The container that wraps the alert message.
+ * @csspart alert__icon - The container that wraps the alert icon.
+ */
+export default class SynValidate extends SynergyElement {
+  static styles: CSSResultGroup = [componentStyles, styles];
+
+  static dependencies = {
+    'syn-alert': SynAlert,
+  };
+
+  controller = new AbortController();
+
+  @queryAssignedElements() private slottedChildren: HTMLElement[];
+
+  @state() validationMessage = '';
+
+  @state() eagerFirstMount = true;
+
+  /**
+   * The variant that should be used to show validation alerts.
+   *
+   * The following variants are supported:
+   * - **native** (default): Uses the native browser validation, usually a browser tooltip.
+   * - **inline**: Show the validation message underneath the element, using a `<syn-alert>`
+   */
+  @property({ reflect: true }) variant: 'native' | 'inline' = 'native';
+
+  /** Do not show the error icon when using the inline variant validation */
+  @property({ attribute: 'hide-icon', reflect: true, type: Boolean }) hideIcon = false;
+
+  /**
+   * Defines the events that trigger the validation.
+   * `invalid` will always automatically be included.
+   * You may also use the `live` keyword to validate on every input change.
+   * `live` will make sure to listen to the `invalid`, `input` and `blur` events.
+   *
+   * Please have a look at the [documentation for native form validation](https://developer.mozilla.org/en-US/docs/Learn/Forms/Form_validation)
+   * and [the use of form invalid events](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/invalid_event) for further information.
+   *
+   * @example ```html
+   * <!-- Validate on invalid and change events (invalid, change) -->
+   * <syn-validate on="invalid change"></syn-validate>
+   *
+   * <!-- Validate on live events (invalid, blur, input)-->
+   * <syn-validate on="live"></syn-validate>
+   *
+   * <!-- Validate on live and custom events (invalid, blur, input, focus, change) -->
+   * <syn-validate on="live focus change"></syn-validate>
+   * ```
+   */
+  @property({ reflect: true }) on: string = '';
+
+  /**
+   * Custom validation message to be displayed when the input is invalid.
+   * Will override the default browser validation message.
+   * Set to an empty string to reset the validation message.
+   */
+  @property({ attribute: 'custom-validation-message', type: String }) customValidationMessage = '';
+
+  /**
+   * Set this to true to validate the input immediately when it is rendered.
+   * Best used with a `variant` of `inline`.
+   * When setting eager, the input will not be focused automatically.
+   *
+   * When using a `variant` of `native` the browser will focus
+   * the last eager field as it is using a tooltip.
+   * In this case it is better to just provide one eager field.
+   */
+  @property({ type: Boolean }) eager = false;
```

Comment:
I tried out setting the `eager` prop after initialization.
I would have expected, that the validation would trigger and show the error message, but nothing happened.

I'm not sure, if this is also the behavior, we really want, but I just wanted to share my expectation on this.
What do you mean, @schilchSICKAG ?

Reply:
You are right, will add this, too 👍

Reply:
Fixed in latest